saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap

```diff
@@ -31084,7 +31084,7 @@ exports[`Storyshots Views / Products / Product list default 1`] = `
             <td
               class="MuiTableCell-root-id MuiTableCell-body-id Component-textRight-id"
             >
-              XAG 274.99
+              XAG274.99
```

Comment:
Looks like you're using earlier than 10 version of node

Reply:
Updated node and snapshots.